### ISSUE TITLE
Make backend printed logs clearer and concise

### ISSUE THREAD

**Issue by we-ai** [episphere/connect#647]
Ideally, the printout logs provide key info about status of our running services, to make it easier to narrow down issues when something is not right. 
Currently, some printout info are too brief, which make it harder to trace back to the source code. And some are too verbose (eg: a printout of Firebase query snapshot), which make logs check harder and may also increase our bills on GCP.
Thus we need to update our backend printout logs and make them clearer and concise.

**Comment by anthonypetersen** [comment]
This probably needs to be looked at on a case by case basis. I don't necessarily agree with the GCP costs as a reason to decrease what we put in our logs. We were charged $0.67 last month total for all services related to Cloud Logging.
 
 In my opinion, with all of the filtering options available within GCP I've never found it overly difficult to narrow down to the information I'm looking for. And coming from the experience of having to troubleshoot bugs that occurred within the backend, more information is better than less.
 
 I still think some things probably don't need to fill up the logs as much as they do (such as printing out the entire SSO object of every person that logs into our applications) but I think we need to be careful about taking potentially useful information out of the logs when viewed from the point of view of troubleshooting.

**Comment by we-ai** [comment]
We're on the same page that logs need to be clear and concise, so that they're can be more helpful. As mentioned above, some of our current logs are too brief, while some are too verbose. Below put examples and possible improvements.

Some logs are too brief, for example `console.log(api)`. There're 2 issues with this example log:

- Without a consistent tag, filtering all logs related to API calls is not easy
- From printed logs, it's hard to trace back to the line of source code that printed them

These issues can be alleviated by adding extra info in the logs, for example `console.log('Called API:', api)`. With the extra info (`Called API:` in above example), it's easier to filter these logs and trace the logs back to source code.  

For logs that are too verbose, we may include key information in logs.  For example, current code print out decrypted SSO and some Firestore query snapshots. In both cases, most parts of each log are not useful. 

In summary, a consistent log format is preferred, for example `console.log(stringForFilteringAndTracing, keyInfoVariable)`.

**Comment by anthonypetersen** [comment]
I think both of these sound like good ideas, particularly adding extra info on console log statements.

**Comment by anthonypetersen** [comment]
If there are specific things that need to be addressed, let's open a different Issue to track the effort. Or else we can track this in a discussion.
